UniConnectApp/app/src/main/java/com/example/uniconnect/service/UniversityService.kt

```diff
@@ -11,8 +11,9 @@ import retrofit2.awaitResponse
 class UniversityService : IUniversityService {
     override suspend fun fetchUniversities(): List<University>? {
         return withContext(Dispatchers.IO) {
-            val retrofit = RetrofitClientInstance.retrofitInstance?.create(IUniversityDAO::class.java)
-            val universities = async { retrofit?.getAllUniversities()}
+            val retrofit =
+                RetrofitClientInstance.retrofitInstance?.create(IUniversityDAO::class.java)
+            val universities = async { retrofit?.getAllUniversities() }
```

Comment:
automatic via kotlin style guide per https://kotlinlang.org/docs/coding-conventions.html#apply-the-style-guide